utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,146 @@
+    [rocky8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the management of
+    hugepages with the [spdk](https://spdk.io/) library.
+
+
+## Building Docker Images
+
+### Building Base DAOS Image
+
+The first image to create is the `daos-base` image which is not intetended to be used as it, but as
+a base image for building the other three daos images.  This first image could be built directly
+from GitHub with the following command:
+
+```bash
+$ docker build --tag daos-base:centos8 \
+	"https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8"
+```
+
+or from a local tree with the following command:
+
+```bash
+$ docker build --tag daos-base:centos8 "utils/docker/vcluster/daos-base/el8"
+```
+
+This Docker file accept the following arguments:
+- `RHEL_BASE_IMAGE`: Base docker image to use (default centos)
+- `RHEL_BASE_VERSION`: Version of the base docker image to use (default 8)
+- `BUST_CACHE`: Manage docker building cache (default undefined).  To invalidate the cache, a random
+	value such as the date of the day shall be given.
+- `DAOS_AUTH`: Enable DAOS authentification when set to "yes" (default "no")
+- `DAOS_REPOS`: List of repos needed to install DAOS (default "https://packages.daos.io/v2.0")
+- `DAOS_GPG_KEYS`: List of GPG keys associated with DAOS repos (default
+	"https://packages.daos.io/RPM-GPG-KEY")
+- `DAOS_REPOS_NOAUTH`: List of repos to use without GPG authentification (default "")
+
+### Building DAOS Nodes Images
+
+The three images `daos-server`, `daos-admin` and `daos-client` could be built directly from GitHub
+or from a local tree in the same way as for the `daos-base` image.
+
+The Docker file of the `daos-server` image accept the following arguments:
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentification when set to "yes" (default "no")
+- `DAOS_HUGEPAGES_NBR`: Number of huge pages to allocate for SPDK (default 4096)
+- `DAOS_IFACE_NAME`: Fabric network interface used by the DAOS engine (default "eth0")
+- `DAOS_SCM_SIZE`: Size in GB of the RAM emulating SCM devices (default 4)
+- `DAOS_BDEV_SIZE`: Size in GB of the file created to emulate NVMe devices (derault 16)
+
+The Dockerfile of the `daos-client` and `daos-admin` images accept the following arguments:
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentification when set to "yes" (default "no")
+- `DAOS_ADMIN_USER`: Name or uid of the daos administrattor user (default "root")
+- `DAOS_ADMIN_GROUP`: Name or gid of the daos administrattor group (default "root")
+
+From a local, tree they could also be built with the following commands:
+
+```bash
+$ cd "utils/docker/vcluster"
+$ docker-compose build
+```
```

Comment:
Could you please also provide the command to build from the GitHub directly?

Reply:
fixed with commit fixed with commit https://github.com/daos-stack/daos/pull/7556/commits/445b4543d7d9ee5f98b5e6de5eca826f0c160b37

Add paragraph containing an example showing how to build these images directly from GitHub.